source/libtomo/gridrec/c/gridrec.c

```diff
@@ -86,10 +86,6 @@ gridrec(const float* data, int dy, int dt, int dx, const float* center,
     if(dy == 0 || dt == 0 || dx == 0)
         return;
 
-#if defined(TOMOPY_CXX_GRIDREC)
-    cxx_gridrec(data, dy, dt, dx, center, theta, recon, ngridx, ngridy, fname,
-                filter_par);
-#else
```

Comment:
Please remove `TOMOPY_CXX_GRIDREC` compiler directives from other cxx and c files too. They are no longer needed now that you have selected files using CMake.